app/resources/v1/advocacy_campaign_resource.rb

```diff
@@ -15,6 +15,8 @@ def target_list
       records.where(action_type: value[0].to_s)
     }
 
+    filter :origin_system  # This enables: http://example.com/advocacy_campaigns?filter[origin_system]=5calls
```

Comment:
I think you can remove this comment.  You've got documentation that would tell API consumers how to get what they need.  I'm not sure if you also need a comment that explains what the code does.

As a general rule, when it comes to adding comments, the code tells you how, comments tell you why.  When you've got code that's doing something unidiomatic or unusual, it's definitely worth adding a comment.   Here, it's probably not necessary.